cli/src/app.rs

```diff
@@ -39,6 +39,11 @@ pub fn app() -> Command {
                 .value_name("FILE")
                 .help("Sets a custom config file")
                 .value_hint(ValueHint::FilePath),
+            Arg::new("no-config")
```

Comment:
I decided to go for a separate option here since `--config /dev/null` would not work on Windows and I see no reason to make something so simple platform-dependent.